tokendito/__init__.py

```diff
@@ -1,7 +1,7 @@
 # vim: set filetype=python ts=4 sw=4
 # -*- coding: utf-8 -*-
 """Tokendito module initialization."""
-__version__ = "2.1.3"
+__version__ = "2.1.4"
```

Comment:
🤔 I am not sure if this should be a bump to the minor or patch versions, so I went with the conservative choice.

Reply:
This has been reverted per request from @pcmxgti.